doc/examples/encryption.rst

```diff
@@ -91,10 +91,34 @@ application from inserting unencrypted values for any fields marked with the
 
 The following example shows how to setup automatic client-side field level
 encryption using
-:class:`~pymongo.encryption.ClientEncryption` to create a new encryption
+:class:`~motor.motor_asyncio.AsyncIOMotorClientEncryption` to create a new encryption
 data key and create a collection with the
 `Automatic Encryption JSON Schema Syntax
 <https://dochub.mongodb.org/core/client-side-field-level-encryption-automatic-encryption-rules>`_.
 
 .. literalinclude:: server_fle_enforcement_example.py
```

Comment:
Can you also update the previous two examples to use AsyncIOMotorClientEncryption instead of ClientEncryption?

Reply:
yep, got it!